src/services/users.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+let users = [];
+
+function getAll() {
+  return [...users];
+}
+
+function getById(id) {
+  if (!id) {
+    return null;
+  }
+
+  const foundUser = users.find(user => String(user.id) === id);
+
+  return foundUser || null;
+}
+
+function create(name) {
+  const newUser = {
```

Comment:
Return a status code 400 if the required name parameter is not passed.

Reply:
Done